backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/scope/model/impl/ElasticsearchScopeModel.java

```diff
@@ -152,4 +153,15 @@ public void checkNestedField(String absoluteFieldPath) {
 			throw log.unknownFieldForSearch( absoluteFieldPath, getIndexesEventContext() );
 		}
 	}
+
+	public Set<String> getNestedDocumentPaths(String absoluteFieldPath) {
```

Comment:
Same comment as for the Lucene backend: I don't think you should support conflicting nested paths in multiple indexes.